main.cpp

```diff
@@ -1,207 +1,127 @@
-#include <Windows.h>
-#include <MMSystem.h>
-#include <iostream>
-#include <io.h>
-#include "dirent.h"
-#include <vector>
+#include "audioplayer.h"
 
-#pragma comment(lib, "winMM.lib")
+void printMenu();
 
-using namespace std;
-
-void getFileNames(const string &, vector<string> *);
-
-string changeVolume(string currentVolume);
-
-string changeTrack(vector<string>);
+int main() {
+    AudioPlayer musicPlayer;
+    bool firstIteration = true;
+    string volume = "50";
 
-void rewindTrack();
+    while (true) {
+        int n;
+        int counter;
+        int trackNumber;
+        string fileLink;
 
-int main() {
-    int n;
-    string trackName, fileLink, currentVolume;
-    vector<string> files;
-    bool isPlaying = false;
+        if (!firstIteration) {
+            // menu
+            printMenu();
 
-    // get file names from folder
-    getFileNames("music/", &files);
+            cin >> n;
 
-    // set initial volume
-    currentVolume = "50";
+            if (isdigit(n) == n) {
+                cout << "\nError. Please enter number";
+                abort();
+            }
 
-    // choose audio file
-    fileLink = changeTrack(files);
+        } else {
+            firstIteration = false;
+            vector<string> files = getFileNames("music/");
 
-    // infinity loop
-    while (true) {
-        // open .mp3 file
-        mciSendString(fileLink.c_str(), nullptr, 0, nullptr);
+            // output files
+            cout << "\nfiles in folder 'music':" << endl;
+            counter = 0;
+            for (auto const file:files) {
+                counter++;
+                cout << counter << ". " << file << endl;
+            }
```

Comment:
Let's create `printAvailableTracks` function in main.cpp

Reply:
We can reuse it here https://github.com/ANANAzZzZz/Music-Player/pull/2/files#diff-608d8de3fba954c50110b6d7386988f27295de845e9d7174e40095ba5efcf1bbR67

Reply:
done